src/autorest-core/app.ts

```diff
@@ -257,43 +257,34 @@ async function currentMain(autorestArgs: string[]): Promise<number> {
   api.GeneratedFile.Subscribe((_, artifact) => artifacts.push(artifact));
   api.ClearFolder.Subscribe((_, folder) => clearFolders.push(folder));
 
-  try {
-    const config = (await api.view);
+  const config = (await api.view);
 
-    // maybe a resource schema batch process
-    if (config["resource-schema-batch"]) {
-      return await resourceSchemaBatch(api);
-    }
+  // maybe a resource schema batch process
+  if (config["resource-schema-batch"]) {
+    return await resourceSchemaBatch(api);
+  }
 
-    // maybe a merge process
-    if (config["merge"]) {
-      return await merge(api);
-    }
+  // maybe a merge process
+  if (config["merge"]) {
+    return await merge(api);
+  }
 
-    if (config["batch"]) {
-      await batch(api);
-    }
-    else {
-      const result = await api.Process().finish;
-      if (result !== true) {
-        throw result;
-      }
+  if (config["batch"]) {
+    await batch(api);
+  }
+  else {
+    const result = await api.Process().finish;
+    if (result !== true) {
+      throw result;
     }
+  }
 
-    // perform file system operations.
-    for (const folder of clearFolders) {
-      try { await ClearFolder(folder); } catch (e) { }
-    }
-    for (const artifact of artifacts) {
-      await WriteString(artifact.uri, artifact.content);
-    }
+  // perform file system operations.
+  for (const folder of clearFolders) {
+    try { await ClearFolder(folder); } catch (e) { }
   }
-  catch (e) {
-    api.Message.Dispatch({
```

Comment:
this catch made no sense. It swallowed `Exceptions` too which are handled correctly by the caller of this method... let's trust our caller.